Explorer/Assets/DCL/SDKComponents/MediaStream/Systems/UpdateMediaPlayerSystem.cs

```diff
@@ -86,7 +86,7 @@ private void UpdateAudioStream(ref MediaPlayerComponent component, PBAudioStream
 
             if (component.State != VideoState.VsError)
             {
-                float actualVolume = sdkComponent.Volume * worldVolumePercentage * masterVolumePercentage;
+                float actualVolume = sdkComponent.HasVolume ? sdkComponent.Volume : MediaPlayerComponent.DEFAULT_VOLUME * worldVolumePercentage * masterVolumePercentage;
```

Comment:
Shouldnt both kind of volumes be multiplied by the percentages?

Reply:
what do you mean?

Reply:
ah sorry, the name might not be the best, the value is a 0->1 float